src/wasm-lib/kcl/src/docs/mod.rs

```diff
@@ -621,6 +633,69 @@ pub fn get_description_string_from_schema(schema: &schemars::schema::RootSchema)
     None
 }
 
+pub fn cleanup_number_tuples_root(mut schema: schemars::schema::RootSchema) -> schemars::schema::RootSchema {
+    cleanup_number_tuples_object(&mut schema.schema);
+    schema
+}
+
+fn cleanup_number_tuples_object(o: &mut schemars::schema::SchemaObject) {
+    if let Some(object) = &mut o.object {
+        for (_, value) in object.properties.iter_mut() {
+            *value = cleanup_number_tuples(value);
+        }
+    }
+
+    if let Some(array) = &mut o.array {
+        if let Some(items) = &mut array.items {
+            match items {
+                schemars::schema::SingleOrVec::Single(_) => {
+                    // Do nothing since its only a single item.
+                }
+                schemars::schema::SingleOrVec::Vec(items) => {
+                    if items.len() == 2 {
+                        // Get the second item and see if its a NumericType.
```

Comment:
Should we create a wrapper type? Jess suggested it might simplify building the docs.